cmd/reindex/main.go

```diff
@@ -41,6 +48,12 @@ type AWSConfig struct {
 	tlsInsecureSkipVerify bool
 }
 
+type DatasetEditionMetadata struct {
+	id        string
+	editionID string
+	version   string
+}
+
 type zebedeeClient interface {
```

Comment:
As mentioned above - it now makes sense to move this to the clients package

Reply:
Fixed in update 1.